identity/src/main/java/com/android/identity/mdoc/response/MobileSecurityObjectGenerator.java

```diff
@@ -14,11 +14,14 @@
  * limitations under the License.
  */
 
-package com.android.identity;
+package com.android.identity.mdoc.response;
```

Comment:
I don't think `mdoc.response` is the right place because it's also something you need on the issuer side... `mdoc.mso` maybe?

Reply:
Also, the tests should be moved as well

Reply:
done